src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,250 @@
+                System.out.println("Please try to add event again υ´• ﻌ •`υ");
+                printLine();
+            } catch (FormatException e) {
+                System.out.println("WOOFS!!! The format of entering event is incorrect.");
+                System.out.println("Please try to add event again υ´• ﻌ •`υ");
+                printLine();
+            }
+            break;
+        case "mark":
+            try {
+                markTask(commandArgs);
+            } catch (NoDescriptionException e) {
+                System.out.println("WOOFS!!! The index of entering task must be stated.");
+                System.out.println("Please try to mark task again υ´• ﻌ •`υ");
+                printLine();
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("WOOFS!!! The index of entering task is not valid.");
+                System.out.println("Please try to mark task again υ´• ﻌ •`υ");
+                printLine();
+            }
+            break;
+        case "unmark":
+            try {
+                unmarkTask(commandArgs);
+            } catch (NoDescriptionException e) {
+                System.out.println("WOOFS!!! The index of entering task must be stated.");
+                System.out.println("Please try to mark task again υ´• ﻌ •`υ");
+                printLine();
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("WOOFS!!! The index of entering task is not valid.");
+                System.out.println("Please try to mark task again υ´• ﻌ •`υ");
+                printLine();
+            }
+            break;
+        case "help":
+            info();
+            break;
+        default:
+            throw new InvalidCommandException();
+        }
+
+    }
+
+    private static String inputCommand() {
+        Scanner scan = new Scanner(System.in);
+        String s;
+        s = scan.nextLine();
+        while (s.trim().isEmpty() || s.trim().charAt(0) == '#') {
+            s = scan.nextLine();
+        }
+        return s;
+    }
+
+    private static void unmarkTask(String commandArgs) throws NoDescriptionException, IndexOutOfBoundsException{
+        if (commandArgs.trim().length() == 0) {
+            throw new NoDescriptionException();
+        }
+        final int unmarkId = Integer.parseInt(commandArgs) - 1;
+        if (unmarkId < 0 || unmarkId >= taskCount) {
+            throw new IndexOutOfBoundsException();
+        }
+        if (!tasks[unmarkId].isDone) {
+            System.out.println("This task hasn't been marked as done yet ∪･ω･∪");
+        } else {
+            tasks[unmarkId].markAsNotDone();
+            System.out.println("I've unmarked this task ∪･ω･∪:");
+            System.out.println(tasks[unmarkId]);
+        }
+        printLine();
+    }
+
+    private static void markTask(String commandArgs) throws NoDescriptionException, IndexOutOfBoundsException{
```

Comment:
`markTask` and `unmarkTask` seems pretty similar. Maybe you can try abstracting them to not repeat yourself.